develop/best_practices/Code Robustness.md

```diff
@@ -0,0 +1,48 @@
+---
+uid: Code_Robustness
+---
+
+# Code Robustness
+
+## Convert.ToString()
+
+- (string)o: Fastest internally, but assumes the object already IS a string. If it isn't, this will throw exceptions.
```

Comment:
Please avoid using contractions on docs: "isn't" should be "is not", "won't" should be "will not", etc.